### DISCUSSION TITLE
Avoid nested fixups by appropriately placing point in the magit-log-select buffer

### DISCUSSION THREAD

**Discussion by farblos** [magit/magit#5188] [Ideas]
During review, I usually create a lot of fixup commits, all on top of some initial commit A:

```
b53d210 * branch-foo-bar-baz fixup! Foo bar baz.
3c5fdd9 * fixup! Foo bar baz.
040cb9a * fixup! Foo bar baz.
ea1590c * fixup! Foo bar baz.
0987581 * fixup! Foo bar baz.
46db5a5 * fixup! Foo bar baz.
945306e * fixup! Foo bar baz.
bb9298b * fixup! Foo bar baz.
f3ad54c * fixup! Foo bar baz.
83030f5 * fixup! Foo bar baz.
c8ec4e7 * Foo bar baz.
```

I don't want to squash these immediately, so `magit-commit-instant-fixup` is not an option here, since I want to keep track of the changes made during review.

For that workflow it would be useful if function `magit-log-select`, when called from `magit-commit-squash-internal`, would place point on the first non-fixup commit to avoid "nested" fixups (`fixup! fixup! fixup! ...`).

For the above example, when I do `c f` to do one more fixup commit, I'd expect point on the line with commit `c8ec4e7` so that I could just `C-c C-c`.

(But then I'm still in my (Ma)git adolescence so I might be missing something here.)

Thanks!

**Comment by tarsius** [comment]
Makes sense, thanks for the suggestion. Implemented in 1fdb30cc1a0860c6b27d982b5bb6329843674357.
